Dockerfile

```diff
@@ -1,7 +1,7 @@
 # DSSG Solve container build file
 #
 
-ARG PYVERSION=3.6.5
+ARG PYVERSION=3.7
```

Comment:
:man_shrugging: v3.7 is cool and doesn't seem to hurt :smile:

Reply:
v3.7 was not out when the project started, it is less than a month old. Coming from the Java world it seems unwise to me to change the version of the environment without some in depth testing, but I don't know enough about how stable Python new releases are to make an informed decision.